rtapi.go

```diff
@@ -382,6 +395,15 @@ func createPDF(endpoints []endpointDetails, output string) {
 	os.Stdout.Write([]byte("PDF report generated successfully!\n"))
 }
 
+func showProgressBar(sum int) {
+	os.Stdout.Write([]byte("Total " + strconv.Itoa(sum) + " seconds.\n"))
```

Comment:
Can you tweak the `Total: etc...` output? I'm thinking of something along the lines of `rtapi will take X seconds to run` might sound a bit better?